mergify_engine/squash_pull.py

```diff
@@ -46,7 +47,7 @@ async def _do_squash(
     try:
         await git.init()
 
-        if ctxt.subscription.active:
+        if subscription.Features.PUBLIC_REPOSITORY:
```

Comment:
To check a feature you need to use the `ctxt.subscription.has_feature()` method.

The feature is called BOT_ACCOUNT